### PR TITLE
[#164982115] Configure VS code for ruby and JS debugging

### PR DESCRIPTION
**Pull request by software-project**
https://www.pivotaltracker.com/story/show/164982115

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,3 +106,18 @@ To run all tests (unit and e2e):
 To run an individual test:
 
 `yarn test:all path/to/test`
+
+## Debugging
+
+You can debug all tests and Rails server in VS code.
+Go to debug view (⇧+⌘+D on Mac)
+Form the dropdown in left top corner pick what you want to debug:
```
Comment (akegan) [inline README.md]:
```suggestion
From the dropdown in left top corner pick what you want to debug:
```

File: README.md [hunk 2]
```diff
@@ -106,3 +106,18 @@ To run all tests (unit and e2e):
...
+- Rails server: for running app
+- Jest: for any javascript test
+- Rspec: for ruby tests
+
+For tests you can debug single file or whole suite. To enter debug click a brakepoint (red dot) left side to line numer of the file you want to debug and run the test with play button. In the debug console you can run any function to check the expected outcome. Also adding watch expressions is very helpful. You can use the bar that popups to jump to next lines of code, into the functions or continue.
```
Comment (akegan) [inline README.md]:
```suggestion
For tests you can debug a single file or the whole suite. To enter debug click a breakpoint (red dot) left side to line number of the file you want to debug and run the test with play button. In the debug console you can run any function to check the expected outcome. Also adding watch expressions is very helpful. You can use the bar that popups to jump to next lines of code, into the functions or continue.
```

**Review by akegan — APPROVED** [review]
looks good just a couple of minor typos

**Comment by akegan** [comment]
@software-project just had a couple of suggested changes on this then you can merge

**Comment by software-project** [comment]
Thanks @akegan

**Pull request merged**